test/unit/generator/content/part2/part2.test.js

```diff
@@ -0,0 +1,109 @@
+const part2 = require('../../../../../app/generator/content/part2')
```

Comment:
need to mock `getReductions`, `getTable` and `mapReductions` for unit test


think you can move this file into integration test

Reply:
I still think it's a unit test.  We've just decomposed the unit into smaller cleaner units.  It doesn't cross any application layers so don't see it as integration.

This one may in time be worth mocking the sub functions if the complexity to build the content grows, but I don't think it's necessary yet.